webapp/components/admin_console/custom_integrations_settings.jsx

```diff
@@ -145,13 +145,13 @@ export default class WebhookSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.overrideTitle'
-                            defaultMessage='Enable webhooks and slash commands to override usernames:'
+                            defaultMessage='Enable integrations to override usernames:'
                         />
                     }
                     helpText={
                         <FormattedMessage
```

Comment:
You'll need to make this `FormattedHTMLMessage` instead of `FormattedMessage` otherwise your link tag won't work